tests/qos/qos_sai_base.py

```diff
@@ -603,7 +606,11 @@ def stopServices(
             Returns:
                 None
         """
-        duthost = duthosts[rand_one_dut_hostname]
+        if 'dualtor' in tbinfo['topo']['name']:
+            duthost = upper_tor_host
```

Comment:
are we always doing the test on upper_tor?

Reply:
Yes